frontend/mr-admin-flate/src/pages/tiltaksgjennomforinger/tilsagn/OpprettTilsagnSkjemaPage.tsx

```diff
@@ -67,17 +78,32 @@ export function OpprettTilsagnSkjemaPage() {
         </Heading>
       </Header>
       <ContainerLayout>
-        <SkjemaContainer>
-          <SkjemaContent>
-            {tiltaksgjennomforing ? (
-              <OpprettTilsagnContainer
-                tiltaksgjennomforing={tiltaksgjennomforing}
-                tilsagnSkalGodkjennes={godkjenningsModus}
-                tilsagn={tilsagn}
-              />
-            ) : null}
-          </SkjemaContent>
-        </SkjemaContainer>
+        <VStack gap={"8"}>
+          <SkjemaContainer>
+            <SkjemaContent>
+              {tiltaksgjennomforing ? (
+                <OpprettTilsagnContainer
+                  tiltaksgjennomforing={tiltaksgjennomforing}
+                  tilsagnSkalGodkjennes={godkjenningsModus}
+                  tilsagn={tilsagn}
+                />
+              ) : null}
+            </SkjemaContent>
+          </SkjemaContainer>
+
+          <div>
+            <Heading size="medium">Aktive tilsagn</Heading>
+            <SkjemaContainer>
+              <SkjemaContent>
+                {tilsagner && tilsagner.length > 0 ? (
+                  <Tilsagnstabell tilsagn={tilsagner} />
```

Comment:
Vi må vel filtrere `tilsagner` til å bare ta de aktive. Ser at annulert tilsagn også er med nå. Du kan også bare kalle variabelen for `tilsagn` siden det blir flertallsformen for alle tilsagnene.
![image](https://github.com/user-attachments/assets/01318307-a90a-47c5-bd05-14f65849ba20)